driver/csiplugin/controllerserver.go

```diff
@@ -752,106 +922,74 @@ func (cs *ScaleControllerServer) CreateVolume(ctx context.Context, req *csi.Crea
 
-		scaleVol.VolName = fmt.Sprintf("%s-T%scsi", scaleVol.VolName, scaleVol.Tier)
-		err = scaleVol.Connector.SetFilesystemPolicy(ctx, &policy, scaleVol.VolBackendFs)
+		dataTierName, err := scaleVol.Connector.GetFirstDataTier(ctx, scaleVol.VolBackendFs)
 		if err != nil {
-			klog.Errorf("[%s] volume:[%v] - setting policy failed [%v]", loggerId, volName, err)
-			return nil, err
+			return status.Error(codes.Unavailable, fmt.Sprintf("tier info request could not be completed: filesystemName %s", scaleVol.VolBackendFs))
 		}
-
-		// Since we are using a SET POOL rule, if there is not already a default rule in place in the policy partition
-		// then all files that do not match our rules will have no defined place to go. This sets a default rule with
-		// "lower" priority than the main policy as a catch all. If there is already a default rule in the main policy
-		// file then that will take precedence
-		defaultPartitionName := "csi-defaultRule"
-		if !scaleVol.Connector.CheckIfDefaultPolicyPartitionExists(ctx, defaultPartitionName, scaleVol.VolBackendFs) {
-			klog.Infof("[%s] createvolume: setting default policy partition rule", loggerId)
-
-			dataTierName, err := scaleVol.Connector.GetFirstDataTier(ctx, scaleVol.VolBackendFs)
-			if err != nil {
-				return nil, status.Error(codes.Unavailable, fmt.Sprintf("tier info request could not be completed: filesystemName %s", scaleVol.VolBackendFs))
-			}
-			defaultPolicy := connectors.Policy{}
-			defaultPolicy.Policy = fmt.Sprintf("RULE 'csi-defaultRule' SET POOL '%s'", dataTierName)
-			defaultPolicy.Priority = 5
-			defaultPolicy.Partition = defaultPartitionName
-			err = scaleVol.Connector.SetFilesystemPolicy(ctx, &defaultPolicy, scaleVol.VolBackendFs)
-			if err != nil {
-				klog.Errorf("[%s] volume:[%v] - setting default policy failed [%v]", loggerId, volName, err)
-				return nil, err
-			}
+		defaultPolicy := connectors.Policy{}
+		defaultPolicy.Policy = fmt.Sprintf("RULE 'csi-defaultRule' SET POOL '%s'", dataTierName)
+		defaultPolicy.Priority = 5
+		defaultPolicy.Partition = defaultPartitionName
+		err = scaleVol.Connector.SetFilesystemPolicy(ctx, &defaultPolicy, scaleVol.VolBackendFs)
+		if err != nil {
+			klog.Errorf("[%s] volume:[%v] - setting default policy failed [%v]", loggerId, volName, err)
+			return err
 		}
 	}
+	return nil
+}
 
-	volReqInProcess, err := cs.IfSameVolReqInProcess(scaleVol)
-	if err != nil {
-		return nil, err
-	}
-
-	if volReqInProcess {
-		klog.Errorf("[%s] volume:[%v] - volume creation already in process ", loggerId, scaleVol.VolName)
-		return nil, status.Error(codes.Aborted, fmt.Sprintf("volume creation already in process : %v", scaleVol.VolName))
-	}
+func (cs *ScaleControllerServer) mapStatusOfDriverCopyJob(ctx context.Context, req *csi.CreateVolumeRequest, volSrc *csi.VolumeContentSource, scaleVol *scaleVolume, isVolSource bool, isSnapSource bool, snapIdMembers scaleSnapId) (*csi.CreateVolumeResponse, error) {
```

Comment:
Rename the function name `mapStatusOfDriverCopyJob` to something like `getCopyJobStatus`?

Reply:
updated the function name.